### PR TITLE
Add display bip47 counterparty in tx list

### PR DESCRIPTION
**Pull request by Overtorment**
image
image

### CODE REVIEW

**Review by marcosrdz — APPROVED** [review]

**Comment by marcosrdz** [comment]
CI failed

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/dn4gzeyfrbxahparjlpxtlbn7i?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/1480)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/h3vrswaxw742sgi5jltj3s2za4?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/1481)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/4j3ebyvfqumxwzsu2tz53y6kfa?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/1483)

**Comment by GladosBlueWallet** [comment]
Wake the fuck up samurai, we have PRs to merge

image

[all PRs for @limpbrains] https://github.com/BlueWallet/BlueWallet/pulls/review-requested/limpbrains

File: loc/en.json [hunk 1]
```diff
@@ -373,6 +373,8 @@
     "status_cancel": "Cancel Transaction",
     "transactions_count": "Transactions Count",
     "txid": "Transaction ID",
+    "from": "From:",
+    "to": "To:",
```
Comment (limpbrains) [inline loc/en.json:377]: It should be "From: {counterparty}". Because word order might be very different for different languages

**Review by limpbrains — CHANGES_REQUESTED** [review]

**Review by limpbrains — APPROVED** [review]

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/4o7wbybcxc6rufyxoqs4zmmbiu?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/1492)

**Comment by GladosBlueWallet** [comment]
Unbelievable. You, [subject name here], must be the pride of [subject hometown here]!

**Pull request merged**